packages/mds-ingest-service/repository/index.ts

```diff
@@ -505,6 +508,29 @@ export const IngestRepository = ReadWriteRepository.Create(
         }
       },
 
+      getH3Bins: async (params: GetH3BinOptions) => {
+        try {
+          const { start, end = now(), h3_resolution, k = K_HOURLY } = params
+
+          const connection = await repository.connect('ro')
+          const entities = await connection
+            .createQueryBuilder()
+            .select(['provider_id'])
+            .addSelect(h3_resolution, 'h3_identifier')
+            .addSelect(`COUNT(${h3_resolution})`, 'count')
+            .from('telemetry_annotations', 'annotation')
+            .where('timestamp >= :start', { start })
+            .andWhere('timestamp <= :end', { end })
+            .groupBy(`provider_id, ${h3_resolution}`)
+            .having(`COUNT(${h3_resolution}) >= :k`, { k })
+            .execute()
```

Comment:
Can `h3_resolution` be parameterized (for example like how `k` is)?

Reply:
Oh, on that note then, you could use the `h3_identifier` alias on line 525 instead of re-declaring `COUNT(${h3_resolution})` if you wanna!